index.html

```diff
@@ -36,7 +36,7 @@ <h2>Let me know about upcoming Tech Commute news!</h2>
                             </div>
                             <div class="optionalParent">
                                 <div class="clear foot">
-                                    <input type="submit" name="subscribe" id="mc-embedded-subscribe" class="button" value="Subscribe">
+                                    <input data-testid="subscribeButton" type="submit" name="subscribe" id="mc-embedded-subscribe" class="button" value="Subscribe">
                                     <p style="margin: 0px auto;"><a href="http://eepurl.com/iDN-uA" title="Mailchimp - email marketing made easy and fun"></a></p>
                                 </div>
                             </div>
```

Comment:
Test ids are a best practice in the test automation world. There are other ways to locate DOM nodes, but this is the most stable.